sdk/eventhub/eventhubs-checkpointstore-blob/test/blob-partition-manager.spec.ts

```diff
@@ -10,7 +10,7 @@ chai.use(chaiString);
 import debugModule from "debug";
```

Comment:
Can we add tests that ensure we save a checkpoint correctly when sequencenumber or offset are 0?

Reply:
Done.